src/frontend/screens/HowToLeaveProject.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from 'react';
+import {StyleSheet, View} from 'react-native';
+import {Text} from '../sharedComponents/Text';
+import {defineMessages, useIntl} from 'react-intl';
+import {Button} from '../sharedComponents/Button';
+import {NativeStackScreenProps} from '@react-navigation/native-stack';
+import {AppStackParamsList} from '../sharedTypes/navigation';
+import {LIGHT_GREY} from '../lib/styles';
+import Warning from '../images/Warning.svg';
+import {ScreenContentWithDock} from '../sharedComponents/ScreenContentWithDock';
+import {useProjectMembers} from '../hooks/server/projects';
+import {useDeviceInfo} from '../hooks/server/deviceInfo';
+import {COORDINATOR_ROLE_ID, CREATOR_ROLE_ID} from '../sharedTypes';
+import {UIActivityIndicator} from 'react-native-indicators';
+import {QuestionMarkWithShadow} from '../sharedComponents/icons/QuestionMarkWithShadow';
+
+const m = defineMessages({
+  howTo: {
+    id: 'screens.HowToLeaveProject.howTo',
+    defaultMessage: 'How to Leave Project',
+  },
+  instructions: {
+    id: 'screens.HowToLeaveProject.instructions',
+    defaultMessage:
+      'To leave this project please uninstall and reinstall CoMapeo. All project data will be removed from this device.',
+  },
+  goBack: {
+    id: 'screens.HowToLeaveProject.goBack',
+    defaultMessage: 'Go back',
+  },
+  warning: {
+    id: 'screens.HowToLeaveProject.warning',
+    defaultMessage:
+      'If you are the only Coordinator on the project no one else will be able to edit project details or invite other devices!',
+  },
+});
+
+export const HowToLeaveProject = ({
+  navigation,
+}: NativeStackScreenProps<AppStackParamsList, 'HowToLeaveProject'>) => {
+  const {formatMessage} = useIntl();
+  const membersQuery = useProjectMembers();
+  const deviceInfo = useDeviceInfo();
+
+  const coordinators = !membersQuery.data
+    ? []
+    : membersQuery.data.filter(
+        member =>
+          member.role.roleId === COORDINATOR_ROLE_ID ||
+          member.role.roleId === CREATOR_ROLE_ID,
+      );
```

Comment:
nit: remove the negation

```suggestion
  const coordinators = membersQuery.data
    ? membersQuery.data.filter(
        member =>
          member.role.roleId === COORDINATOR_ROLE_ID ||
          member.role.roleId === CREATOR_ROLE_ID,
      );
    : []
```